wheatley/row_generation/helpers.py

```diff
@@ -18,7 +20,11 @@
     "caters": 9,
     "royal": 10,
     "cinques": 11,
-    "maximus": 12
+    "maximus": 12,
+    "thirteen": 13,
```

Comment:
I believe that `Thirteen` is called `Septuples` and `Fifteen` is called `Sextuples` (see [`Stedman Sextuples`](https://rsw.me.uk/blueline/methods/view/Stedman_Sextuples) and [`Stedman Septuples`](https://rsw.me.uk/blueline/methods/view/Stedman_Septuples)).

However, we don't force this mapping to be uniquely invertible so we could just make `thirteen` and `septuples` both map to `13`.  I'm not sure if there'd be any strange repercussions to that?

Reply:
Oops I think I was just looking at the bell mapping and missed these aren't just like Fourteen/Sixteen